### ISSUE TITLE
Split-screen the same app

### ISSUE THREAD

**Issue by weiluntong** [oasisfeng/island#457]
Phone: Oneplus 9 Pro LE2127 (T-mobile variant) 
OS: OxygenOS 13.1 LE2127_13.1.0.522(EX01) (Android 13)

Expected Result:
Split the top and bottom screen real-estate between two apps (specifically two of the same app with one in the work profile and one in the personal profile)

Actual Result:
When selecting a work profile app, then starting the procedure to split the screen between that app and it's personal profile counterpart, the phone will just refocus on the work profile app. And the same happens visa versa (when starting from the personal profile app and selecting the work profile equivalent).

Steps to Reproduce:
1. Ensure Firefox is cloned
2. Open the work profile Firefox
3. Open the personal profile Firefox 
4. Open the current open apps overview
5. Navigate to the work profile Firefox instance
6. Press the 3 dots button (kebab menu button)
7. Press "Split screen" 
8. Open the current open apps overview
9. Press the personal profile Firefox

Notes:
These steps can be performed with any combination of work profile apps and personal profile apps that do not share the app name. In cases which the apps do not share a name, the screen properly splits the screen between the two apps, regardless of if it's a work or personal profile app.

**Comment by weiluntong** [comment]
I've since found out that an interesting workaround for this, at least in OOS 13.1, is to open one as a floating window, then open the other. Following, you can drag the floating window to the top to snap it much like Windows 11 gestures. From there it'll automatically resize the current active app such that the two apps are now split-screen and sharing vertical space.

TL;DR steps for OOS 13.1 workaround:
1. Open work profile Firefox
2. Open app overview
3. Locate work profile Firefox and press kabob menu button
4. Press "Floating Window"
5. Open personal profile Firefox
6. Drag floating Window of work profile Firefox to the top and hover until a snap animation (much like Windows 11 Window snapping gestures) plays. From there, just release the floating Window.

This should result in the work profile Firefox taking the top half of the screen and the personal profile Firefox taking the bottom half.

I guess to note, I'm not sure what the implications of sandbox violations this may cause. Since I've installed my keyboard on my work profile, I actually can't tell  which keyboard is being used XD regardless of which though I guess either one technically violates the sandboxing rules as now, if it's the isolated keyboard, then it's gathering data on the personal profile app split on the bottom half, and if it's the personal profile keyboard, it's gathering data on the work profileapp on the top half... I'm not entirely an expert on isolated environments past using them to create reproduce able builds for software, so I'm not sure if this is actually a concern or not. I guess another thing I've observed is the difficulty adjusting my volume (laggy input response after pressing the hardware button) with this split-screen which might suggest that there's *some* isolation happening? Or maybe that'sbad. Maybe there some conflicting response to the hardware input which is causing the lag. I will say returning to the home screen before adjusting my volume immediately restores the response time to the hardware input. Then I just return to the split-screen apps from the Apps Overview and continue on with my thing.

**Comment by aballa442** [comment]
My app is not work
